substantial/backends/redis.py

```diff
@@ -11,70 +9,133 @@
 class RedisBackend(Backend):
     def __init__(self, host: str, port: int, **kwargs):
         self.redis = redis.Redis(host=host, port=port, decode_responses=True, **kwargs)
+        self.separator = ":/"
```

Comment:
any specific reason for 2 chars?

Reply:
I had issues with `:` only, also a single character as a separator has higher risk of unintentionally splitting meaningful parts of a key (e.g., `2024-01-01 00:00:00` when using `sep=':'`).
I wanted to keep  `:` as it is supported by many Redis client browsers like [this one](https://github.com/joeferner/redis-commander) for grouping the keys so I just added `/`.